lib/commands/deploynodeapp.js

```diff
@@ -495,14 +696,29 @@ function handleUploadResult(err, req, fileName, itemDone) {
 
 // Create a target endpoint that references the Node.js script
 function createTarget(opts, request, done) {
-  var targetDoc = mustache.render(
+  var targetDoc;
+  
+  // we need the XML for a HostedTarget (or whatever)...this is just a place holder that isn't a ScriptTarget
+  if (opts.runtime === EdgeServicesRuntimeOption) {
+    targetDoc = mustache.render(
+    '<TargetEndpoint name="default">' +
```

Comment:
We used the `ScriptContainer` tag which I believe pulled the routing key from the KVM, so that will be the same functionally...will it have to target a different KVM map? I forget how that actually worked. It also targeted the LB in front of our cluster for us...not sure we can just drop it in. Is there a specific change you'd like to see here? I'm not sure what to do without an actually policy to use, but happy to do whatever.